src/MoBi.Presentation/Presenter/SelectBuildingBlocksForExtendPresenter.cs

```diff
@@ -0,0 +1,74 @@
+using System.Collections.Generic;
+using System.Linq;
+using MoBi.Assets;
+using MoBi.Core.Domain.Model;
+using MoBi.Core.Domain.Repository;
+using MoBi.Presentation.DTO;
+using MoBi.Presentation.Views;
+using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Presentation.Presenters;
+
+namespace MoBi.Presentation.Presenter
+{
+   public interface ISelectBuildingBlocksForExtendPresenter : IDisposablePresenter
+   {
+      IEnumerable<MoleculeBuildingBlock> GetMolecules();
```

Comment:
Is the underlying prop IEnumerable or IReadonly?

Reply:
They were IEnumerable, but they will become IReadOnly